Formula/n/node.rb

```diff
@@ -97,54 +86,6 @@ def install
 
     system "./configure", *args
     system "make", "install"
-
-    # Allow npm to find Node before installation has completed.
-    ENV.prepend_path "PATH", bin
-
-    bootstrap = buildpath/"npm_bootstrap"
-    bootstrap.install resource("npm")
-    # These dirs must exists before npm install.
-    mkdir_p libexec/"lib"
-    system "node", bootstrap/"bin/npm-cli.js", "install", "-ddd", "--global",
-            "--prefix=#{libexec}", resource("npm").cached_download
-
-    # The `package.json` stores integrity information about the above passed
-    # in `cached_download` npm resource, which breaks `npm -g outdated npm`.
-    # This copies back over the vanilla `package.json` to fix this issue.
-    cp bootstrap/"package.json", libexec/"lib/node_modules/npm"
-    # These symlinks are never used & they've caused issues in the past.
-    rm_r libexec/"share" if (libexec/"share").exist?
-
-    bash_completion.install bootstrap/"lib/utils/completion.sh" => "npm"
-  end
-
-  def post_install
```

Comment:
Note that removing this will likely mean that user-installed Node modules go into the 🗑️ when they `brew upgrade node`.

Reply:
I had no idea, and this was not intentional. What chunks of the `post_install` cover that?